### PR TITLE
Prototype Pollution in shvl

### PR DESCRIPTION
**Pull request by gavin105**
I would like to report a **Prototype pollution vulnerability** in **shvl**.

It fails to restrict access to prototypes of objects, allowing for modification of prototype behaviour, which may allow obtaining sensitive information/DoS/RCE.

### CODE REVIEW

**Comment by gavin105** [comment]
By mistake I have created multiple commits to make sure only one vulnerability got reported in this PR. Sorry for the inconvenience.

**Review by mzfr — APPROVED** [review]
LGTM!

image

File: bounties/npm/shlv/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "Summary": "Prototype Pollution",
+    "Contributor": {
+        "Discloser": "Gavin Matthew",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "shvl",
+        "URL": "https://www.npmjs.com/package/shvl",
+        "Downloads": "433132/month"
```
Comment (Mik317) [inline bounties/npm/shlv/1/vulnerability.json:14]: Please remove `/month` :+1:

**Review by Mik317 — APPROVED** [review]
LGTM :smile: :cake:

Cheers,
Mik

File: bounties/npm/shlv/1/vulnerability.json (hunk 1 above)
Comment (gavin105) [reply bounties/npm/shlv/1/vulnerability.json:14]: I have deleted my huntr repository due to multiple commits while submitting my next vulnerability disclosure and I re-forked it again. Due to this I am unable to make the modification to this file. Can you please do the modification on behalf of me. Thank you very much in advance.

**Comment by JamieSlome** [comment]
@gavin105 - this PR looks good. Unfortunately, the JSON is invalid and the directory name is mispelled - should be `shvl`.

I am going to close this. Feel free to re-open a new PR with the corrections and we will merge it in.

Make sure to reference this PR in your new PR...

Cheers! 🍰

**Pull request closed without merging**